files/lib/system/exporter/WBB4xExporter.class.php

```diff
@@ -2181,6 +2246,16 @@ public function exportBlogEntries($offset, $limit)
         }
     }
 
+    private function getCoverPhotoPath(string $blogFilePath, array $row): string
+    {
+        $coverPhotoID = $row["coverPhotoID"];
```

Comment:
```suggestion
        $coverPhotoID = $row['coverPhotoID'];
```
Same in the lines below.